pkg/jaeger/store/trace_query.go

```diff
@@ -15,66 +15,93 @@ import (
 )
 
 const (
+	// A lateral join is used for the `link` table instead of using directly
```

Comment:
can we please add a note about why we need a group by to the comment. It's a bit counter-intuitive so would be good to document

Reply:
I had automerge already enabled, I'll add it in a follow up PR